README.md

```diff
@@ -33,7 +33,7 @@ Require `bllim/laravel-validation-for-client-side` in composer.json and run `com
 
     {
         "require": {
-            "laravel/framework": "5.0.*",
+            "laravel/framework": "5.1.*",
```

Comment:
I haven't tested with 5.0.
I have used 5.1 so i do not have any idea.
i think it will work in 5.0 too but i am not 100% sure.